src/patternfly/components/Table/table.scss

```diff
@@ -200,6 +200,31 @@
     table-layout: fixed;
   }
 
+  &.pf-m-sticky-header {
+    position: relative;
+
+    > thead > tr {
+      border-bottom: 0;
+
+      > * {
+        position: sticky;
+        top: 0;
+        z-index: var(--pf-global--ZIndex--xs);
+        background: var(--pf-c-table--BackgroundColor);
```

Comment:
why is this necessary?

Reply:
it gives the header cells a background so that the contents of the table don't show through as you scroll
![Jun-17-2020 15-13-48](https://user-images.githubusercontent.com/35148959/84945696-30a26000-b0ad-11ea-833a-981b10464a20.gif)

Reply:
oh awesome!